homeassistant/components/sma_manager/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""THE SMA Manager integration."""
+
+# Home Assistant Imports
+# 3rd Party Imports
+from sma_manager_api import SMA
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+# Local Imports
+from .const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_REFRESH_INTERVAL, DOMAIN
+
+PLATFORMS: list[str] = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, config: ConfigEntry) -> bool:
+    """Configure the Integration.
+
+    @param hass:
+    @param config:
+    @return:
+    """
+
+    # Save to SMA object to state machine
```

Comment:
Please don't add comments like these, explaining *what* obvious code does.
Add comments which explain *why* we do something if not obvious, or *what* the code does if the code is complicated. In this case, no comment is needed.